book/src/offline-signing/durable-nonce.md

```diff
@@ -0,0 +1,201 @@
+# Durable Transaction Nonces
+
+Durable transaction nonces are a mechanism for getting around the typical
+short lifetime of a transaction's [`recent_blockhash`](../transaction.md#recent-blockhash).
+They are implemented as a Solana Program, the mechanics of which can be read
+about in the [proposal](../implemented-proposals/durable-tx-nonces.md).
+
+## Known Issues
+
+### Fee Theft Opportunity
+
+The durable nonce implementation contains a vulernability which allows for fees
+to be stolen by a transaction using the feature under certain conditions. To
+achieve this, the transaction must have failed with an instruction error, which
+rolls back the step that advanced the stored nonce, allowing the transaction to
+be replayed and fees charged. This can be repeated until the stored nonce is
+successfully advanced.
+
+#### - Mitigation
+
+To minimize loss of funds, a low-balance account should always be used to pay
+fees on a durable nonce transaction.
+
+If a transaction using the durable nonce feature fails with an instruction error,
+a new transaction, which only advances the nonce must be submitted to the network
```

Comment:
Missing a comma after "advances the nonce"

Reply:
Does this strictly need to be a new-nonce-only transaction? It could be any transaction that includes the new-nonce instruction, right?

So maybe something like this would be better:
"If a transaction using the durable nonce feature fails with an instruction error, submit a new transaction that advances the nonce immediately. The easiest way to do this is with a single-instruction NonceInstruction::Nonce transaction." <-- perhaps the CLI command is better than referring to the Nonce program here, but something like this...

Reply:
Doesn't strictly need to be new-nonce-only, no.  I clarified that and integrated your suggestion